File: latihan2/python/product.py
class product():
    # atribut
    __price = 0
    __idProduct = "blank"

    # constructor
    def __init__(self, price=0, idProduct="blank"):
        self.__price = price
        self.__idProduct = idProduct

    # Setter dan Getter

    def setPrice(self, price):
        self.__price = price
        
    def getPrice(self):
        return self.__price 

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct
        
    def getIdProduct(self):
        return self.__idProduct 

File: latihan2/php/product.php
<?php
    class product{
        // atribut
        private $price = 0;
        private $idProduct = "blank";
     
        // constructor
        function __construct($price=0, $idProduct="blank"){
            $this->price = $price;
            $this->idProduct = $idProduct;
        }

        // Setter dan Getter

        function setPrice($price){
            $this->price = $price;
        }
        function getPrice(){
            return $this->price;
        }

        function setIdProduct($idProduct){
            $this->idProduct = $idProduct;
        }
        function getIdProduct(){
            return $this->idProduct;
        }

        // destructor
        function __destruct(){}

    }
?>